list2/ex47/index.js

```diff
@@ -0,0 +1,16 @@
+export function transformingObjectProperties() {
+  const data = {
+    a: 1,
+    b: 2,
+    c: 3,
+  };
+
+  const data2 = {};
+
+  const multiplier = Object.entries(data);
+  multiplier.forEach(([key, value]) => {
+    data2[key] = value * 2;
+  });
+  console.log(JSON.stringify(data2));
+}
+transformingObjectProperties();
```

Comment:
I think your solution does not solve what is being asked in the exercise. Please, revisit it and make sure your code does what is being asked.